### PR TITLE
Dsh/week6 api rails

### PR DESCRIPTION
**Pull request by dHelmgren**
## Description
Making sure we got any rails specific stuff out of this resource.

### CODE REVIEW

**Comment by dHelmgren** [comment]
[Remove Rails from testing APIs lesson](https://trello.com/c/e3J6cb7T/1342-remove-rails-from-testing-apis-lesson)

**Comment by dHelmgren** [comment]
[Remove Rails from testing APIs lesson](https://trello.com/c/e3J6cb7T/1342-remove-rails-from-testing-apis-lesson)

File: 02-intermediate-ruby/api-testing-vcr.md [hunk 1]
```diff
@@ -23,23 +21,15 @@ Each interaction is recorded as a **cassette**. We can load cassettes in our tes
...
 
 ### Installation
-1. **Gem**
-
-  There are a few gems that we want to include to test using VCR. First, we'll load those into our `development` and `test` sections.
-
-  ```ruby
-  group :development, :test do
-    gem 'minitest-vcr'
-    gem 'webmock'
```
Comment (kaidamasaki) [inline 02-intermediate-ruby/api-testing-vcr.md]: Should these be replaced with something like below?
```bash
$ gem install minitest-vcr
$ gem install webmock
$ gem install dotenv
```

Or even:

```bash
$ gem install minitest-vcr webmock dotenv
```

File: 02-intermediate-ruby/api-testing-vcr.md [hunk 2]
```diff
@@ -69,7 +59,7 @@ Each interaction is recorded as a **cassette**. We can load cassettes in our tes
   it "Can send valid message to real channel" do
     VCR.use_cassette("channels") do
       message = "test message"
-      response = SlackApiWrapper.send_msg("<CHANNELID>", message)
+      response = SlackApiWrapper.send_msg("<CHANNELID>", message)
```
Comment (kaidamasaki) [inline 02-intermediate-ruby/api-testing-vcr.md]: Accidental whitespace?

File: 02-intermediate-ruby/api-testing-vcr.md [hunk 3]
```diff
@@ -89,8 +79,18 @@ Each interaction is recorded as a **cassette**. We can load cassettes in our tes
 
 Once you run a test that uses VCR, you'll notice that there will be a file in the `test/cassettes` folder with the name corresponding to the parameter provided in the `use_cassette` method.
 
-If you expect the response data to change, you must delete the cassette file.
+If you expect the response data to change, _you must delete the cassette file._
```
Comment (kaidamasaki) [inline 02-intermediate-ruby/api-testing-vcr.md]:
```suggestion
If you expect the response data to change, _**you** must delete the cassette file._
```

File: 02-intermediate-ruby/api-testing-vcr.md [hunk 4]
```diff
@@ -89,8 +79,18 @@ Each interaction is recorded as a **cassette**. We can load cassettes in our tes
...
+
+Our process for testing with VCR is as follows:
+
+1. Ensure that we are using `dotenv` to hide any personal information or secrets that we might have to send as part of a test
+
+1. Write and run our tests as normal
+
+1. Check to see that our cassettes have been instansiated.
+
+1. When a test needs to change, delete the existing cassette and proceed from step 2.
```
Comment (kaidamasaki) [inline 02-intermediate-ruby/api-testing-vcr.md]: If we're going to refer to a specific section do we want to use h3s so we can link to it?

**Pull request merged**